tests/utils/verification/VolumePropertiesVerificationUtil.go

```diff
@@ -90,6 +98,30 @@ func GetVolumePropertiesDockerCli(volName string, hostname string) string {
 	return op
 }
 
+//GetVolumeStatus - get the volume status on a given host
+func GetVolumeStatus(name, ip string) string {
+	cmd := DOCKER_CLI_INSPC + " --format '{{index .Status.status}}' " + name
+	op := ExecCmd(ip, cmd)
+	return strings.TrimSpace(op)
+}
+
+// PollStatusDetached - check if the status gets detached within the timeout
+func PollStatusDetached(name, ip string, timeout int) bool {
```

Comment:
Can we improve this API to make it generic to support polling both attached and detached status?

Moreover, I'd prefer not to expose the timeout parameter, because it will not only simplify the API client, but also avoid changing multiple places if the default timeout value turns out inappropriate (too long or too short).